components/generic/LinkGroup.vue

```diff
@@ -21,7 +21,17 @@
           :destination="link.url"
           :link-class="linkClass"
         >
-          {{ link.text }}
+          <template v-if="link.icon">
+            <span class="icon-background">
+              <i
+                :class="link.icon"
+                :aria-label="link.text"
```

Comment:
Should this be on the `title` attribute instead so that it's visible on hover?

Reply:
I have changed this into a title.